docker-compose.yml

```diff
@@ -64,11 +64,22 @@ services:
       - ./setup.cfg:/usr/src/app/setup.cfg
       - ./pytest.ini:/usr/src/app/pytest.ini
       - ./tox.ini:/usr/src/app/tox.ini
+      - ./scripts:/usr/src/app/scripts
     environment:
       BIGCHAINDB_DATABASE_BACKEND: mongodb
       BIGCHAINDB_DATABASE_HOST: mdb
       BIGCHAINDB_DATABASE_PORT: 27017
       BIGCHAINDB_SERVER_BIND: 0.0.0.0:9984
+      BIGCHAINDB_GRAPHITE_HOST: graphite
     ports:
       - "9984"
-    command: bigchaindb start
+    command: bash -c 'bigchaindb -y configure mongodb && bigchaindb start'
```

Comment:
**Note**: PR #1517 improves the Dockerfile build step so that it is not only mongodb specific, but also supports rethinkdb.